assignment-2/7s.md

```diff
@@ -1,2 +1,91 @@
+unique and beautiful, instead of focusing on the price of the tickets. Instead of nickle-and-diming their customers for drinks and food like many other airlines, they prefer to just include it in the ticket price. This gives a more luxurious and relaxed feeling during flights. Together with this they also aim for a very user friendly
+atmosphere by focusing a great deal of attention to twitter and facebook customer service.
+##### Structure
+As to be expected of a 32.000 employee company, the structure is very classical and hierarchical.
+The executive committe consists out of 13 people [5], each responsible for their subdivison of KLM. This kind of structure is not the best for quick decision making, but it is good at managing very large companies. And tons of independent scrum teams consisting out of all the personnel needed to maintain, supply, staff and fly a plane would most likely become a big mess. Companies like this need big planning departments and overarching management to make decisions on what course of action to take.
+##### Systems
+The advantage of hierarchical systems is that it is usually very easy to identify the chain of command. Within the airplane everyone answers to the Captain and after that to the second in command. Who again answer to the chief of staff for the type of airplane they are flying. Due to the large amount of pilots (2.600) they rarely see the chief of staff though.  The advantage of this is, that it is easy to get things done, when on a trip the Captain is in charge of all decisions and only when something goes seriously wrong, you have to see your manager.
+##### Staffing
+A main pro of being a company this big and well known, with a prestigous brand name is that it is not very hard to get the right staff. People know how to find KLM, and this shows. They usually have no open vacancies for aircraft crew. Most open vacancies are usually in the IT field.
+##### Skills
+As noticed before, a main skill of KLM is customer engagement through Twitter and Facebook, this helps in creating a very friendly brand name. Other skills of note are the international networking and cooperation through the Skyteam network, ensuring that you can reach virtually any destination of the world while flying with KLM or one of their partners.
+##### Style
+KLM's leadership style is very detached from the 'workfloor'. Captains all have their own style of leadership in their airplane. The main way of influence from higher management is through flight plans and regulations. But as stated earlier, pilots really only see their supervisor once something goes really wrong. So if a captain flies slightly faster than planned, which burns more fuel, to be on their destination earlier that will likely not attract any attention at all.  This means that the area in which leadership has the most influence is in all the supporting acitivities: routes, staff planning, catering, hotels, management, cargo and training.
```

Comment:
It would be really interesting if you were able to expand a bit how the leadership works inside the plane in the strategy section. It sounds very interesting and there are a few details like if the captain chooses the routes or if the crew is a team that always works together that would give an extra deepness to the report.